packages/styles/src/tile.scss

```diff
@@ -767,15 +767,23 @@ $fd-line-tile-badge-horizontal-spacing: 0.375rem !default;
     width: auto;
     padding-block: 0;
     padding-inline: $fd-line-tile-horizontal-padding;
-    box-shadow: none;
-    background: transparent;
+    box-shadow: var(--sapContent_Shadow0);
+    background: var(--sapTile_Background);
+    border-radius: var(--sapTile_BorderCornerRadius);
+    border: var(--sapTile_BorderColor);
+    display: flex;
+    gap: 0.5rem;
 
     @include fd-hover() {
-      box-shadow: none;
+      box-shadow: var(--sapContentShadow2);
+      background: var(--sapTile_Hover_Background);
     }
 
     @include fd-active() {
       box-shadow: none;
+      background: var(--sapTile_Active_Background);
+      outline-offset: -0.125rem;
+      border: 2px solid var(--sapContent_FocusColor);
```

Comment:
border-width should be in `rem` value: 0.125rem(2px)

Reply:
done!